packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -813,6 +814,12 @@ class TextStyle with Diagnosticable {
   /// How visual text overflow should be handled.
   final TextOverflow? overflow;
 
+  /// The string used to ellipsize text when [overflow] is set to
+  /// [TextOverflow.ellipsis]. Otherwise it's ignored.
+  /// Defaults to U+2026 HORIZONTAL ELLIPSIS (…).
```

Comment:
we should have a blank line before line 819, so that it gets separated into paragraphs in the output.

Reply:
(otherwise dartdoc just jams it all together into one line)